docs/layers/s2-layers/s2-layer.md

```diff
@@ -0,0 +1,93 @@
+<p class="badges">
+  <img src="https://img.shields.io/badge/64--bit-support-blue.svg?style=flat-square" alt="64-bit" />
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
+</p>
+
+# S2Layer (Experimental)
+
+The S2Layer renders filled and/or stroked polygons, with geometry automatically calculated based on an S2 token (geospatial index).
```

Comment:
Recommend adding a link to http://s2geometry.io/ the first time it is mentioned on a page.